doc/developers/contributing.rst

```diff
@@ -40,6 +40,20 @@ See :ref:`new_contributors` to get started.
     others of the Python Software Foundation:
     https://www.python.org/psf/codeofconduct/
 
+Our Code of Conduct
+===================
+Here is a summary of our Code of Conduct:
+
+* Be respectful of differing viewpoints and experiences. Harrassment, insults, and threats are not tolerated.
+* Accept constructive criticism peacefully. When you disagree, please be courteous in raising your issues.
+* Do not publish others' private information, such as a physical or electronic address, without explicit permission.
+* Do not incite violence, including encouraging a person to commit suicide or engage in self-harm.
+* Do not insult or hold others up for ridicule.
+
+If you engage in behavior that violates this Code of Conduct, we may take any action we deem appropriate, including warnings or expulsion from the community.
```

Comment:
Not sure we can enforce that in any way TBH

Reply:
While I would not add anything regarding the summary because I would like to use the PSF COC, we might want to add guidelines on how issues would be resolved which we don't have for the moment. However, we should discuss how to handle those when I am looking at other projects.

I think that it would be best to be added there: https://www.scipy.org/codes-of-conduct.html